lib/frame_window_filtering/hog_labeling.h

```diff
@@ -0,0 +1,54 @@
+/// \file
+/// Maintainer: Felice Serena
+///
+
+#pragma once
+
+#include "frame_window_filtering.h"
+
+#include "classifier/classifier.h"
+
+#include <memory>
+
+namespace MouseTrack {
+
+class HogLabeling : public FrameWindowFiltering {
```

Comment:
Would you really consider this a filtering operation?

Reply:
Yes. "Filtering" is probably a bad term, I think of it as an operation that takes a frameWindow and returns a modified copy. As always, open for a better terminology.